Pages/aboutUs.js

```diff
@@ -0,0 +1,12 @@
+class AboutUs {
+    constructor(){
+        this.title = "About Us"
+    }
+    get aboutParagraph(){
+        return browser.driver.findElement(by.css(".helper div h2"));
+    }
+    getAboutParagraph(){
```

Comment:
Fo every method add some console log which will explain what is happening in that method. Test will be easier to debug with this.

Reply:
I added console.log() in every method